### ISSUE TITLE
documentation

### ISSUE THREAD

**Issue by suzuki-hoge** [suzuki-hoge/has-hub#22]

**Comment by suzuki-hoge** [comment]
![2018-08-12 17 57 41](https://user-images.githubusercontent.com/18749992/44000339-f062c830-9e58-11e8-9c9c-5b2d666a9165.png)

**Comment by suzuki-hoge** [comment]
build and release note.

**Comment by suzuki-hoge** [comment]
completion (bash update)

**Comment by suzuki-hoge** [comment]
```zsh
autoload -U +X compinit && compinit
autoload -U +X bashcompinit && bashcompinit
eval "$(stack --bash-completion-script stack)"
```

**Comment by suzuki-hoge** [comment]
publish bin only, not stack app.

**Comment by suzuki-hoge** [comment]
fixed #71 !!!
